apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -13,3 +13,46 @@
     }
   }
 }
+
+.reaction-recorder {
+  .webcam {
+    width: 15%;
+
+    video {
+      width: 100%;
+    }
+
+    &.offline {
+      video {
+        display: none;
+      }
+    }
+
+    &::after {
+      content: ' ';
+      display: block;
+      visibility: hidden;
+      background-color: red;
+      width: 0;
+      height: 2px;
+      transition: width 2.5s;
+      transition-timing-function: linear;
+    }
+
+    &.recording {
+      &::after {
+        visibility: visible;
```

Comment:
Properties should be ordered width, visibility

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)